clang/lib/AST/Interp/Opcodes.td

```diff
@@ -582,11 +598,21 @@ def CastIntegralFloating : Opcode {
 
 // Cast a floating to an integer type
 def CastFloatingIntegral : Opcode {
-  let Types = [AluTypeClass];
+  let Types = [FixedSizeIntegralTypes];
```

Comment:
The type classes for these get kinda confusing... but I just can't come up with the correct syntax to define them inline.